dev/server.js

```diff
@@ -42,6 +42,12 @@ createServer(async (req, res) => {
     return req.pipe(proxyReq, { end: true })
   }
 
+
+  if(path.startsWith('/assets/')){
+    const file = await readFile(join(rootDir,path));
+    res.writeHead(200,{'Content-Type':getContentType(getExtname(path))})
+   return res.end(file)
+  }
```

Comment:
here i catch every path that begin by "/assets/" like a static file , i use the function ``getExtname``to get the file extension and the function ``getContentType``to get the Content-type